jpsreport/scripts/plot_geom.py

```diff
@@ -1,62 +0,0 @@
-import numpy as np
```

Comment:
Is there an alternative with the jupyter notebooks?

Reply:
yes, see the description of this PR.

Reply:
sorry, I can't find any description about plotting geometry. What do you mean in detail?

Reply:
@chraibi Was the file _plot_geom.py_ replaced by the file _helpers/Utilities.py_ (commit bcf8a4ff)?